01/app.js

```diff
@@ -0,0 +1,31 @@
+const form = document.querySelector("form")
+
+
+if (form) {
+    form.addEventListener('submit', handleSubmit)
+
+    function addItemToList(user) {
+        const list = document.querySelector('ul')
+        if (list) {
+            list.appendChild(user)
+        }
+    }
+
+    function createUserItem(firstName, lastName) {
+        const item = document.createElement('li')
+        item.classList.add('users-list__person')
+        item.textContent = firstName + ' ' + lastName
+        addItemToList(item)
+    }
+
+    function handleSubmit(event) {
+        event.preventDefault()
+
+        const firstName = form.elements[0].value
+        const lastName = form.elements[1].value
+
+        if (firstName.trim() && lastName.trim()) {
+            createUserItem(firstName, lastName)
+        } else alert("Niepoprawne dane")
+    }
```

Comment:
Może lepiej wrzucić te funkcję poza `if` - będzie czytelniej :)

Dodatkowo lepiej to zrobić od ogółu do szczegółu, aby wygodnie się czytało.
Teraz najwyżej jest funkcja, która jest uruchamiana w kodzie tj. `handleSubmit`, potem mam deklarację tej funkcji, która jest wykorzystywana w `handleSubmit` tj. ` createUserItem`, a na końcu `addItemToList`.

Jak to ktoś stwierdził  - teraz można czytać kod jak poezję ;P

```
const form = document.querySelector("form")


if (form) {
    form.addEventListener('submit', handleSubmit)
}


function handleSubmit(event) {
    event.preventDefault()

    const firstName = form.elements[0].value
    const lastName = form.elements[1].value

    if (firstName.trim() && lastName.trim()) {
        createUserItem(firstName, lastName)
    } else alert("Niepoprawne dane")
}


function createUserItem(firstName, lastName) {
    const item = document.createElement('li')
    item.classList.add('users-list__person')
    item.textContent = firstName + ' ' + lastName
    addItemToList(item)
}

function addItemToList(user) {
    const list = document.querySelector('ul')
    if (list) {
        list.appendChild(user)
    }
}
```